config/rbac/role.yaml

```diff
@@ -10,18 +10,19 @@ rules:
   - eno.k8s.io
   resources:
   - connectionpoints
+  - l2services
```

Comment:
This file is automatically generated when you run the make deploy command. So is this changes have happened through make deploy or by hand ?

Reply:
This is discussed with the Team. The changes are automatically generated

Reply:
We havent modified it by hand.